Euclidean_Space.R

```diff
@@ -0,0 +1,183 @@
+#Euclidean Space R
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+
+# """
+# Euclidean space.
+# """
+#
+# import geomstats.backend as gs
+#
+# from geomstats.manifold import Manifold
+# from geomstats.riemannian_metric import RiemannianMetric
+#
+#
+# class EuclideanSpace(Manifold):
+#   """
+# Class for Euclidean spaces.
+#
+# By definition, a Euclidean space is a vector space of a given
+# dimension, equipped with a Euclidean metric.
+# """
+
+
+EuclideanSpace <- setRefClass("EuclideanSpace",
+                              contains = Manifold,
+                              methods = list(
+
+                                # def __init__(self, dimension):
+                                #   assert isinstance(dimension, int) and dimension > 0
+                                # self.dimension = dimension
+                                # self.metric = EuclideanMetric(dimension)
+
+                                init <- function(self,dimension){

```

Comment:
Spacing: see Google style  convention for R:
https://google.github.io/styleguide/Rguide.xml#filenames

-> `self, dimension`